test/unit/application_test.rb

```diff
@@ -66,6 +66,11 @@ def setup
     end
   end
 
+  test 'load returns nil if application does not exist' do
+    app = Application.load('1003', '2003')
```

Comment:
In the file I see a method named Test::Unit::TestCase method defined (https://ruby-doc.org/stdlib-1.8.7/libdoc/test/unit/rdoc/Test/Unit/TestCase.html#method-i-setup)

That performs a flush of the database and resets the memoizer. I understand this would ensure that no Application/Service/any content exists at the beginning of each test.

Reply:
ah, that could work, yes